### PR TITLE
List of forbidden words

### PR DESCRIPTION
**Pull request by alqahtani**
Hi,
thanks for the great app. I think I'll use it a lot :).
although It's should be a 50/50 chance when you ask the app, but some topics should not be joked about. like suicidal questions etc. I remember that there was a bot on twitter replies to  "should I" questions with yes all the time. but they also tried their best to avoid that type of questions, and they have some great list of responses to them as well.

I hope you find my added feature is helpful 👍 

ahmed

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -11,10 +14,22 @@ exports.api = async () => {
...
 };
 
+exports.absolutelyNo = (text) => {
+  for (let i=0; i<this.forbiddenWords.length; i++) {
+    if ( text.includes(this.forbiddenWords[i])) {
+      return true;
+    }
+  }
+  return false;
+}
```
Comment (cedricium) [inline src/utils/index.js]: Needs a semi-colon after the `}`.

File: src/utils/index.js [hunk 2]
```diff
@@ -11,10 +14,22 @@ exports.api = async () => {
...
-exports.yesno = () => {
+exports.yesno = (fullQuestion) => {
+  if (this.absolutelyNo(fullQuestion)) {
+    return 'no'
+  }
   return Math.floor(Math.random() * 2) === 0 ? 'yes' : 'no';
 };
 
+exports.absolutelyNo = (text) => {
+  for (let i=0; i<this.forbiddenWords.length; i++) {
```
Comment (cedricium) [inline src/utils/index.js]: Add spacing between the operators, i.e. `let i = 0; i < this...`.

File: src/utils/index.js [hunk 3]
```diff
@@ -42,3 +57,10 @@ exports.yesSayings = [
   'yeah',
   'yes',
 ];
+
+exports.forbiddenWords = [
+  'kill myself',
+  'commit suicide',
+  'take my own life',
+  'rob a'
+];
```
Comment (cedricium) [inline src/utils/index.js]: Add a newline to the end of this file.

**Review by cedricium — CHANGES_REQUESTED** [review]
Hey @alqahtani, I'm glad you like `should-i-cli` and thanks so much for the PR! I definitely agree protective censoring would be a good idea and I think you've set this feature up beautifully.

Just a few minor changes to be made then I'd love to accept this 👍

**Comment by alqahtani** [comment]
thanks for your response.
all mentioned changes are fixed now.

**Review by cedricium — APPROVED** [review]

**Comment by cedricium** [comment]
🚀 merged, thanks again @alqahtani! Will release v1.2.0 on npm in a few minutes.

**Pull request merged**